posthog/auth.py

```diff
@@ -68,8 +69,13 @@ def authenticate(cls, request: Union[HttpRequest, Request]) -> Optional[Tuple[An
             )
         except PersonalAPIKey.DoesNotExist:
             raise AuthenticationFailed(detail=f"Personal API key found in request {source} is invalid.")
-        personal_api_key_object.last_used_at = timezone.now()
-        personal_api_key_object.save()
+
+        now = timezone.now()
+        if personal_api_key_object.last_used_at is None or now - personal_api_key_object.last_used_at > timedelta(
+            hours=12
```

Comment:
What about starting with 1? Not sure if someone is using this for reporting

Reply:
This isn't used anywhere significant - only on FE.